Solutions/Ais.Net.Specs/Ais.Net.Specs.csproj

```diff
@@ -80,10 +80,13 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>NmeaLineToAisStreamAdapterSpecs.feature</DependentUpon>
     </Compile>
-    <Compile Update="Ais\Net\Specs\NmeaStreamParserSpecs.feature.cs">
+    <Compile Update="Ais\Net\Specs\NmeaStreamParserByLineSpecs.feature.cs">
+      <DependentUpon>NmeaStreamParserByLineSpecs.feature</DependentUpon>
```

Comment:
Should this also have the `AutoGen` and `DesignTime` blocks?

Reply:
I have to admit I'm not really sure whether it's strictly necessary—the status of code gen in SpecFlow seems to be an ever-moving target. But I've pushed a commit to at least make this consistent with all the others.